modules/serverless-logic-overview-callbacks.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+// * about/serverless-logic-overview.adoc
+
+
+:_content-type: CONCEPT
+[id="serverless-logic-overview-callbacks_{context}"]
+= Callbacks in Serverless Logic
```

Comment:
The term `Serverless Logic` is repeated several times with each heading. Better to remove the term `Serverless Logic` from all headings because assembly title is sufficient to set up the context that we are talking about SL. Also, It would be helpful to create an attribute for the term Serverless Logic. It is being used quite frequently.